recipe/meta.yaml

```
@@ -45,7 +45,7 @@ requirements:
     - py-lief
     - pyyaml
     - ripgrep
-    - setuptools
+    - setuptools <60.0.0
```

Can we add a comment here:
```suggestion
    # setuptools >=60.0.0 brings its own local distutils, which bdist_conda does not work with
    # see also https://setuptools.pypa.io/en/latest/history.html#v60-0-0
    - setuptools <60.0.0
```

---

Or even adopt the solution in https://github.com/conda-forge/conda-build-feedstock/pull/175

---

Added the build steps from #175